metabase/metabase
# Task to send emails warn user about pulse removal

Adds a task that runs only once to send emails to admins and warn about pulse removal in 49.
We'll send emails iff the instance has at least one active legacy pulse.

How to test:
1. checkout master, make sure you have email setup
2. Create a legacy pulse by going to `{{site-url}}/pulse/create`
3. check out this branch and start the server
4. open maildev and you'll see an email like the screenshot below
5. try shutdown metabase, then restart it, you shouldn't get another email
<img width="694" alt="Screenshot 2023-12-05 at 20 23 29" src="https://github.com/metabase/metabase/assets/25661381/1ea0ff40-b17c-4f9c-b175-6c39812ff960">

Product doc: https://www.notion.so/metabase/Remove-Pulses-while-warning-customers-about-it-781d66840a4d4e109c03bce8e0da3e39?pvs=4

Epic https://github.com/metabase/metabase/issues/34147
